CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - Cleaned up draw function files
 - Removed C4 defuse restriction for teammates
 - Moved role specific score variables into the role base
+- Refactored `GM:HUDDrawTargetID` for the use of a shared FindAlongRay-Function
```

Comment:
i definitely would put the `targetid` library under added, alongside the `FindEntityAlongRay` function shortly mentioning their uses.

also i'm not sure if you need to specify that `GM:HUDDrawTargetID` uses the new library, to who would it matter? not necessarily a question for you but for the other team members

Reply:
Nobody answered but it makes sense to me. Nobody needs to know about the refactor as genreal functionalities didn't change, but there is now a new library. So I added an entry under Added :)